pkg/util/validation/limits.go

```diff
@@ -275,6 +276,7 @@ func (l *Limits) RegisterFlags(f *flag.FlagSet) {
 	f.BoolVar(&l.ServiceOverloadStatusCodeOnRateLimitEnabled, "distributor.service-overload-status-code-on-rate-limit-enabled", false, "If enabled, rate limit errors will be reported to the client with HTTP status code 529 (Service is overloaded). If disabled, status code 429 (Too Many Requests) is used. Enabling -distributor.retry-after-header.enabled before utilizing this option is strongly recommended as it helps prevent premature request retries by the client.")
 	f.BoolVar(&l.OTelMetricSuffixesEnabled, "distributor.otel-metric-suffixes-enabled", false, "Whether to enable automatic suffixes to names of metrics ingested through OTLP.")
 	f.BoolVar(&l.OTelCreatedTimestampZeroIngestionEnabled, "distributor.otel-created-timestamp-zero-ingestion-enabled", false, "Whether to enable translation of OTel start timestamps to Prometheus zero samples in the OTLP endpoint.")
+	f.Var(&l.PromoteOTelResourceAttributes, "distributor.promote-otel-resource-attributes", "Eventual OTel resource attributes to promote to labels.")
```

Comment:
What does "eventual" mean in this context? Is the functionality not implemented?

Reply:
I guess "eventual" is not the best word to use here; my use of it as more or less a synonym for possible (which stems from Norwegian I suppose) is apparently [archaic in English](https://www.merriam-webster.com/dictionary/eventual).

Reply:
Revised the flag help.